gcp/cloud-functions/refresh-pga-data/main.py

```diff
@@ -779,10 +808,16 @@ def query_scorecard_v3(tournament_id, player_id):
                         help="Whether to write results to DB or only print results",
                         action=argparse.BooleanOptionalAction,
                         default=False)
+    parser.add_argument("-a",
```

Comment:
add arg to backfill all tournaments (usually just reprocessed a whitelist passed in, or the current/previous tournament if no arg passed in. I ran this earlier today to populate all tournament data, which took well over an hourl